Dockerfile.wolfi

```diff
@@ -1,3 +1,3 @@
-FROM docker.elastic.co/wolfi/chainguard-base@sha256:9f940409f96296ef56140bcc4665c204dd499af4c32c96cc00e792558097c3f1
+FROM docker.elastic.co/wolfi/chainguard-base:latest@sha256:43569fb7643245f22b17735822618ad67dc1a0ee6dc08862381b4d417b54b969
```

Comment:
Does having label and digest, `name:label@digest`, mean `docker build` will fail when the "latest" label is updated on the the source Docker register to a new digest?

Reply:
Should be fine, inferring from (a) this working

```
docker pull docker.elastic.co/wolfi/chainguard-base:lookatmeimbroken@sha256:9f940409f96296ef56140bcc4665c204dd499af4c32c96cc00e792558097c3f1
```

and (b) the Dockerfile reference https://docs.docker.com/reference/dockerfile/#from slightly unclearly saying:

> The builder returns an error if it can't find the `tag` value.